src/PageContent/Calendar/EventCard/EventCardSlot.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import EventCard from './EventCard';
+import { DeliveryType, StringFormat } from '../../../Enums';
+import Dialog from '../Dialog.js';
+import moment from 'moment';
+
+class EventCardSlot extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            dialogOpen: false
+        };
+
+        this.openDialog = this.openDialog.bind(this);
+        this.closeDialog = this.closeDialog.bind(this);
+    }
+
+    openDialog() {
+        this.setState({
+            dialogOpen: true
+        });
+    }
+
+    closeDialog() {
+        this.setState({
+            dialogOpen: false
+        });
+    }
+
+    render() {
+        if (Object.keys(this.props.events).length === 0) {
+            return <div />;
+        } else {
+            for (var e in this.props.events) {
```

Comment:
it looks like you do create multiple eventcards for multiple events, so you just need to make them show on as stacked vertically (they're probably on top of each other right now)